code/modules/shuttle/on_move.dm

```diff
@@ -134,16 +138,14 @@ All ShuttleMove procs go here
 
 /************************************Turf move procs************************************/
 
-/turf/open/afterShuttleMove(turf/oldT) //Recalculates SSair stuff for turfs on both sides
+/turf/open/afterShuttleMove(turf/oldT, turf_type, baseturf_type, rotation)
 	. = ..()
-	SSair.remove_from_active(src)
-	SSair.remove_from_active(oldT)
-
-	src.CalculateAdjacentTurfs()
-	oldT.CalculateAdjacentTurfs()
-
-	SSair.add_to_active(src, TRUE)
-	SSair.add_to_active(oldT, TRUE)
+	addtimer(CALLBACK(src, /atom/.proc/air_update_turf, TRUE), 0)
+	addtimer(CALLBACK(oldT, /atom/.proc/air_update_turf, TRUE), 0)
+	/* This causes air to leak instead
+	air_update_turf(TRUE)
+	oldT.air_update_turf(TRUE)
+	*/
```

Comment:
@MrStonedOne @duncathan This is the last proc called on the turf after everything else has sorted itself out.

Reply:
removed commented code please